lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -739,4 +740,46 @@ sub estimated_gtmp_for_instrument_data  {
     );
 }
 
+
+sub get_supersede_merged_alignment_results {
```

Comment:
Oops, I got that exactly backwards!  How about `get_superseding_merged_alignment_results`?